packages/fast-components-react-base/src/image/image.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Image",
+    "description": "An image component's schema definition.",
+    "type": "object",
+    "id": "image",
+    "properties": {
+        "alt": {
+            "title": "HTML alt attribute",
+            "type": "string"
+        },
+        "itemScope": {
+            "title": "HTML itemScope attribute",
+            "type": "string"
+        },
+        "round": {
+            "title": "Round",
+            "type": "boolean"
+        },
+        "sizes": {
+            "title": "HTML sizes attribute",
+            "type": "string"
+        },
+        "src": {
+            "title": "HTML src attribute",
+            "type": "string"
+        },
+        "srcSet": {
+            "title": "HTML srcSet attribute",
+            "type": "string"
+        },
+        "vp1": {
+            "title": "Image address at viewport 1",
```

Comment:
Shorten to "Viewport 1 image".

Same with all below

Reply:
fixed